build/Stride.sln

```diff
@@ -332,7 +328,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stride.NuGetResolver", "..\
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuGetResolver", "NuGetResolver", "{158087CF-AF74-44E9-AA20-A6AEB1E398A9}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Stride.Core.Presentation", "..\sources\presentation\Stride.Core.Presentation\Stride.Core.Presentation.csproj", "{0C63EF8B-26F9-4511-9FC5-7431DE9657D6}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stride.Core.Presentation", "..\sources\presentation\Stride.Core.Presentation\Stride.Core.Presentation.csproj", "{0C63EF8B-26F9-4511-9FC5-7431DE9657D6}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Stride.Importer.3D", "..\sources\tools\Stride.Importer.3D\Stride.Importer.3D.csproj", "{66EFFDE4-24F0-4E57-9618-0F5577E20A1E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TEST_BLENDSHAPES", "..\..\..\Stride Projects\TEST_BLENDSHAPES\TEST_BLENDSHAPES\TEST_BLENDSHAPES.csproj", "{C08D307E-7B20-4947-84A7-8ACB0E259518}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TEST_BLENDSHAPES.Windows", "..\..\..\Stride Projects\TEST_BLENDSHAPES\TEST_BLENDSHAPES.Windows\TEST_BLENDSHAPES.Windows.csproj", "{28F346A3-0E96-4C9C-B8FD-8F699FEA21CB}"
```

Comment:
Missing projects ?

Reply:
Accidental checked in, should be fix!

Reply:
Did you forget to commit your changes in ? Or did you mean to say that you've seen this issue and will fix it ?

Reply:
fixed